### ISSUE TITLE
Matching algorithm issues

### ISSUE THREAD

**Issue by dufferzafar** [gsamokovarov/jump#24]
My `scores.json` file is here: https://gist.github.com/60c5f00c08b5769ed99152dbdef9a3da

Some sample lines from the file: (notice how bitcoin has pretty higher score)

```json
{ "Path": "/home/dufferzafar/dev/bitcoin", "Score": {"Weight":48, "Age":"2018-09-03T15:37:08.64200953+05:30"} },
{ "Path": "/home/dufferzafar/.apps/OpenSubtitlesDownload", "Score": {"Weight":14, "Age":"2018-09-03T15:37:08.64200953+05:30"} },
{ "Path": "/home/dufferzafar/dev/@yard/@mtech/sem-1/github-analytics", "Score": {"Weight":14, "Age":"2018-09-03T15:37:08.64200953+05:30"} },
```

Now, 

`jump cd bit` matches with: `/home/dufferzafar/.apps/OpenSubtitlesDownload`

`jump run btc` matches with: `/home/dufferzafar/dev/@yard/@mtech/sem-1/github-analytics`

I expected both of these to match with `bitcoin` :disappointed:

**Comment by gsamokovarov** [comment]
Aaah, did you imported those from autojump with the importer script? The ordering in the `scores.json` files matters as I have optimized `jump cd` to depend on it. It seems that the values are not sorted. Have you integrated jump in the shell? I'm pretty sure the fill will be sorted if you enter a new value into it with `jump chdir` or regular `cd` if you have integrated jump into the shell.

**Comment by gsamokovarov** [comment]
I have released `0.21.0` with `jump import`. You can try it if you wanna setup `jump` anew.

Regards!

**Comment by dufferzafar** [comment]
I can confirm that this has been fixed in 0.21.0
